src/mapkit.cs

```diff
@@ -769,12 +769,15 @@ interface MKMapViewDelegate {
 		[Export ("mapViewDidChangeVisibleRegion:")]
 		void DidChangeVisibleRegion (MKMapView mapView);
 	}
-		
+
```

Comment:
Small, but i would revert changes like this to keep the blame more neat

Reply:
thanks TJ! I'm not sure how to do this now, but will be sure to look out for these in the future

Reply:
Will land it because I don't want to wait for a build, but you can do something like this:
```bash
git diff -U0 -w --no-color | git apply --cached --ignore-whitespace --unidiff-zero -
```